scheduler/integration-test/src/scheduler.test.js

```diff
@@ -54,7 +59,7 @@ describe('Scheduler', () => {
   })
 
   test('GET /jobs', async () => {
-    await sleep(2000) // wait until scheduler does sync
+    await sleep(4000) // wait until scheduler does sync
```

Comment:
This is not actually related to the notification service right? Did this test fail with the shorter delay?

Reply:
Jep